spec.html

```diff
@@ -6241,20 +6254,36 @@
     </emu-clause>
   </emu-clause>
 
+  <emu-clause id="sec-continents">
+    <h1>Continents</h1>
+
+    <p>A <dfn id="continent">continent</dfn> is an independent unit of ECMAScript computation, and is the largest unit that this specification deals with. Everything within this specification is assumed to occur within a single continent.</p>
+
+    <p>The host environment must create a continent prior to the execution of any Jobs or the evaluation of any ECMAScript code.</p>
+
+    <p>A continent may contain multiple realms, as defined in <emu-xref href="#sec-code-realms"></emu-xref>. Each realm belongs to exactly one continent. Depending on the host environment, code executed within a given continent's realms may or may not be able to reach objects from other realms in that continent. However, code executed in one continent must not be able to directly reference or share objects from another continent.</p>
```

Comment:
I agree there's a bit of a contradiction here. I'd argue it's OK, maybe with a slight tweak to the first sentence. In general it seems OK for this sentence to "overreach" a bit, using undefined terms to convey an intention. The sentence is almost entirely drawn from @allenwb's https://esdiscuss.org/topic/event-loops-in-navigated-away-from-windows#content-17

But, happy to remove, especially if it'll help get things in faster.